packages/beacon-node/src/eth1/eth1DepositDataTracker.ts

```diff
@@ -88,7 +88,6 @@ export class Eth1DepositDataTracker {
     this.depositsCache = new Eth1DepositsCache(opts, config, db);
     this.eth1DataCache = new Eth1DataCache(config, db);
     this.eth1FollowDistance = config.ETH1_FOLLOW_DISTANCE;
-    // TODO Electra: fix scenario where node starts post-Electra and `stopPolling` will always be false
```

Comment:
Is this addressed, from a quick look at the PR it seems we only stop polling at the first epoch transition, does this mean there might be a short time after starting the node that it might still try to fetch eth1?

Reply:
Yes, it will run for an epoch. I don't believe it's a big deal though

Reply:
was thinking we could stop polling on startup when we initialize the chain, the `anchorState` should be a finalized state and can check it there

https://github.com/ChainSafe/lodestar/blob/798203114f03e4c2313279b53c6adbfe981be021/packages/beacon-node/src/chain/chain.ts#L258